cvat-ui/src/components/file-manager/file-manager.tsx

```diff
@@ -129,6 +129,8 @@ export default class FileManager extends React.PureComponent<Props, State> {
 
     private renderShareSelector(): JSX.Element {
         function renderTreeNodes(data: TreeNodeNormal[]): JSX.Element[] {
+            // sort alphabetically
+            data.sort((a, b) => (a.key > b.key) ? 1 : -1)
```

Comment:
Static linter shows the issue here: ``Arrow function used ambiguously with a conditional expression.eslintno-confusing-arrow``

Maybe use such kind of sorting?
``data.sort((a: TreeNodeNormal, b: TreeNodeNormal): number => a.key.localeCompare(b.key));``

Reply:
I changed it to your suggestion.